app/webpacker/components/RegistrationsV2/RegistrationAdministration/RegistrationAdministrationList.jsx

```diff
@@ -501,17 +501,16 @@ export default function RegistrationAdministrationList({ competitionInfo }) {
   return (
     <Segment loading={isMutating} style={{ overflowX: 'scroll' }}>
       <Form>
-        <Form.Group widths="equal">
+        <Form.Group unstackable widths="2">
```

Comment:
SemUI is not very good at responsive forms. Good news: Chakra will handle this one million times better. Bad news: There is no better solution within our current constraints.

One alternative that I can think of is `Form.Group inline widths="equal">`. This will still stack the toggles on mobile, but because of the `inline` tag they're using much less space.

Reply:
I think I tried that, but they were literally touching each other, which wasn't enough space to accurate tap one with a thumb. I can try again tomorrow though.

Reply:
That happens when you do `inline` *and* `unstackable`, because then they are not stacked, and the line breaks are super-duper narrow when using `inline`. If you allow them to stack however (by dropping the `unstackable`) then SemUI adds whitespace.

Reply:
Oh, I tried that too. (The `widths` doesn't seem to make any difference.) I think it's a similar amount of space as my current solution? I don't like it as much since there's a big empty space, but if you think it's better then I'll change it.
![inline2](https://github.com/user-attachments/assets/9fe30859-4551-4a0d-9db1-21d10053395a)
![inline1](https://github.com/user-attachments/assets/85539ac9-ad8f-4112-9162-83c6e36beeec)

Reply:
Okay I'll take your solution :D